File: FBOLinx.Web/FBOLinx.Web/ClientApp/src/app/models/PolicyAndAgreementDocuments.ts
import { DocumentAcceptanceFlag, DocumentTypeEnum } from "../enums/documents..enum";
import { PolicyAndAgreementGroupExemptions } from "./PolicyAndAgreementGroupExemptions";
import { UserAcceptedPolicyAndAgreements } from "./UserAcceptedPolicyAndAgreements";

export interface PolicyAndAgreementDocuments {
    oid: number;
    documentType: DocumentTypeEnum;
    documentVersion: string;
    document: string;
    acceptanceFlag: DocumentAcceptanceFlag;
    isEnabled: boolean;
    policyAndAgreementGroupExemptions: PolicyAndAgreementGroupExemptions[];
    userAcceptedPolicyAndAgreements: UserAcceptedPolicyAndAgreements[];
}

File: FBOLinx.Web/FBOLinx.DB/Models/PolicyAndAgreementDocuments.cs
using FBOLinx.Core.Enums;
using System.Collections.Generic;

namespace FBOLinx.DB.Models
{
    public class PolicyAndAgreementDocuments : FBOLinxBaseEntityModel<int>
    {
        public DocumentTypeEnum DocumentType { get; set; }
        public string DocumentVersion { get; set; }
        public string Document { get; set; }
        public DocumentAcceptanceFlag AcceptanceFlag { get; set; }
        public bool IsEnabled { get; set; }
        public List<PolicyAndAgreementGroupExemptions> PolicyAndAgreementGroupExemptions { get; set; }
        public List<UserAcceptedPolicyAndAgreements> UserAcceptedPolicyAndAgreements { get; set; }


    }
}
